portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -113,11 +124,13 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     // Private variables which hold comment data
     private String author;
     private String message;
+    private String email;
     
     // This constructor has two parameters: author and message.
-    public Comment(String a, String m) {
+    public Comment(String a, String m, String e) {
```

Comment:
I suggest making these names author, message, and email and when setting them, do

`this.author = author;`